src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -66,7 +66,7 @@ public function leave(\Twig_Profiler_Profile $profile)
     public function getFunctions()
     {
         $profilerDump = function (\Twig_Environment $env, $value, $maxDepth = 0) {
-            return $value instanceof Data ? $this->dumpData($env, $value, $maxDepth) : twig_escape_filter($env, $this->dumpValue($value));
+            return $value instanceof Data ? $this->dumpData($env, $value, $maxDepth) : $this->dumpValue($value);
```

Comment:
this is wrong. The ValueExporter does not provide safe output (it does not handle escaping)